magda-web-client/src/Components/Dataset/MetadataExtraction/extractExtents.ts

```diff
@@ -28,24 +33,24 @@ export function extractExtents(input, output) {
     }
 }
 
-import XLSX from "xlsx";
-import { uniq } from "lodash";
-const chrono = require("chrono-node");
+moment.tz.setDefault("UTC");
 
-const DATE_REGEX_PART = ".*(date|dt|decade|year).*";
+const DATE_REGEX_PART = ".*(date|dt|year|decade).*";
```

Comment:
Maybe we could pull regexes for determining which columns have dates into config as well, so that it can be changed to support unusual naming schemes.

Reply:
Done